superflore/generators/bitbake/run.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from superflore.generators.bitbake.gen_packages import generate_installers
+from superflore.generators.bitbake.ros_meta import ros_meta
+import shutil
+import sys
+import os
+
+# Modify if a new distro is added
+active_distros = ['indigo', 'kinetic', 'lunar']
```

Comment:
I think I want to wait for Jade EOL to do this, because I don't want to add all the Jade ebuilds to the ros-overlay.